types/account.d.ts

```diff
@@ -146,6 +146,7 @@ export type UpdateCustomerPayload = {
   firstName?: string;
   lastName?: string;
   externalData?: ExternalData;
+  consents?: CustomerConsent[];
```

Comment:
Does the inplayer sdk work the same way that cleeng does where partial updates are allowed without any problems?  That is, can  I just call with `{firstName: 'New Name'}` in order to just update the firstName and leave all other field the same?

Reply:
yes, only `firstName` is required field

Reply:
@dbudzins sorry, it is `PUT` request, and we need to pass all user's data, but still, only `fullName` is required

Reply:
Probably to be safest and most flexible, we should pass in the 'current user data' and then keep this payload just for the update data. Then each cleeng and inplayer service implementation can do internally whatever they need to do with the data.

Reply:
yes, something like this maybe: https://github.com/jwplayer/ott-web-app/blob/e740cdc4ef5099d78d89e7724aa67e6ea8ae0168/src/stores/AccountController.ts#L110

Reply:
this is also improved in the commits below